plugins/modules/openssh_cert.py

```diff
@@ -128,6 +130,13 @@
                Time will always be interpreted as UTC. Mainly to be used with relative timespec for I(valid_from) and / or I(valid_to).
                Note that if using relative time this module is NOT idempotent."
         type: str
+    ignore_timestamps:
```

Comment:
This should also be mentioned in the documentation of `regenerate` (for `partial_idempotence`).